server/src/main/java/io/druid/client/HttpServerInventoryView.java

```diff
@@ -294,18 +295,11 @@ public DruidServer getInventoryValue(String containerKey)
   }
 
   @Override
-  public Iterable<DruidServer> getInventory()
+  public Collection<DruidServer> getInventory()
   {
-    return Iterables.transform(
-        servers.values(), new Function<DruidServerHolder, DruidServer>()
-        {
-          @Override
-          public DruidServer apply(DruidServerHolder input)
-          {
-            return input.druidServer;
-          }
-        }
-    );
+    return servers.values().stream()
```

Comment:
I *think* this change from lazy to eager allocation, but I don't think it will be an issue here.

Reply:
A lot of calls actually use iterations over the inventory. In the near future it might make sense to have this method return a `Stream` so the caller can filter as needed.

Reply:
Yeah, I guess eager allocation is not an issue because the return value is iterated right away in all places.